kpatch-build/create-diff-object.c

```diff
@@ -2705,8 +2686,9 @@ static void kpatch_include_debug_sections(struct kpatch_elf *kelf)
 		if (!is_rela_section(sec) || !is_debug_section(sec))
 			continue;
 		list_for_each_entry_safe(rela, saferela, &sec->relas, list)
-			if (!rela->sym->sec->include)
+			if (!rela->sym->include || !rela->sym->sec->include) {
```

Comment:
Hm, why haven't we seen this problem on x86?

Reply:
Yes, I checked the compiled c file fs/aio.c of x86_64.It reference to __dyndbg  using the __dyndbg + offset,it did not use the 
symbols in the __dyndbg section ,so it has no problem.
But LoongArch generate relas using the symbols in __dyndbg(or __verbose in old version kernel) and triggers the problem.
I think we should avoid that condition even if there is no problem now for x86_64.